sources/Application/AppWindow.cpp

```diff
@@ -457,6 +457,8 @@ void AppWindow::onUpdate() {
   Flush();
 };
 
+void AppWindow::onClockTick() { _currentView->OnClockTick(); }
```

Comment:
Should we call it "AnimationUpdate" or something like that for a generalized use case?

Reply:
@democloid I've pushed a commit which renames the function